okhttp/src/main/java/okhttp3/internal/proxy/NullProxySelector.kt

```diff
@@ -13,27 +13,25 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package okhttp3.internal.proxy;
+package okhttp3.internal.proxy
 
-import java.io.IOException;
-import java.net.Proxy;
-import java.net.ProxySelector;
-import java.net.SocketAddress;
-import java.net.URI;
-import java.util.Collections;
-import java.util.List;
+import java.io.IOException
+import java.net.Proxy
+import java.net.ProxySelector
+import java.net.SocketAddress
+import java.net.URI
 
 /**
- * A proxy selector that always returns the {@link Proxy#NO_PROXY}.
+ * A proxy selector that always returns the [Proxy.NO_PROXY].
  */
-public class NullProxySelector extends ProxySelector {
-  @Override public List<Proxy> select(URI uri) {
+open class NullProxySelector: ProxySelector() {
+  override fun select(uri: URI?): List<Proxy> {
     if (uri == null) {
```

Comment:
nit: `requireNotNull(uri) { "uri must not be null" }`

Reply:
7024be2b154951432f4b79131441b007674a19fe